components/stories/StoryDetail.tsx

```diff
@@ -28,6 +35,7 @@ export default function StoryDetail({ story, onClose }: Props) {
           <Flex>
             <Heading
               as="h2"
+              visibility={categoryLabel[story.category] ? 'visible' : 'hidden'}
```

Comment:
Should this be storyCategoryLabel like below? Or is not supposed to show if its hitting the fallback of `Other`

Reply:
Good spot. Admittedly, this isn't obvious. Yeah, it's supposed to not show if it's Other. But I still want the non-empty label in the hidden div to create the same vertical space as if there _was_ a label there.

Reply:
Coolio, I suspected as much, but figured I would ask!